app/controllers/declaration/DestinationCountryController.scala

```diff
@@ -67,9 +67,9 @@ class DestinationCountryController @Inject()(
 
   private def redirectToNextPage(mode: Mode)(implicit request: JourneyRequest[AnyContent]): Result =
     request.declarationType match {
-      case DeclarationType.SUPPLEMENTARY =>
+      case DeclarationType.SUPPLEMENTARY | DeclarationType.CLEARANCE =>
         navigator.continueTo(controllers.declaration.routes.LocationController.displayPage(mode))
-      case DeclarationType.STANDARD | DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL | DeclarationType.CLEARANCE =>
+      case DeclarationType.STANDARD | DeclarationType.SIMPLIFIED | DeclarationType.OCCASIONAL =>
```

Comment:
Do we need to skip the back end mapping for this too?

Reply:
What do you mean @ed0906 ?

Reply:
The back end is set up to map clearance requests like  standards is it not?
If we are skipping pages, does that mean we can remove that part of the mapping from the back end for this journey type?

Reply:
I think that those fields are optional in the model, so it's not a problem that backend model has everything